tests/test262/expectations.txt

```diff
@@ -1021,7 +1017,6 @@ test/language/statements/let/global-use-before-initialization-in-declaration-sta
 test/language/statements/let/global-use-before-initialization-in-declaration-statement.js(strict mode)
 test/language/statements/let/global-use-before-initialization-in-prior-statement.js(default)
 test/language/statements/let/global-use-before-initialization-in-prior-statement.js(strict mode)
-test/language/statements/let/syntax/identifier-let-allowed-as-lefthandside-expression-non-strict.js(default)
 test/language/statements/let/syntax/with-initialisers-in-statement-positions-label-statement.js(default)
 test/language/statements/let/syntax/with-initialisers-in-statement-positions-label-statement.js(strict mode)
 test/language/statements/let/syntax/without-initialisers-in-statement-positions-label-statement.js(default)
```

Comment:
I'm adding this at the end, but applies to all changes in this file...


EXCELLENT.